### ISSUE TITLE
Small subset of filter names cause exception

### ISSUE THREAD

**Issue by JBeckwith** [tlambert03/fpbasepy#3]
* fpbasepy version: 0.1.1
* Python version: 3.10.12
* Operating System: Ubuntu 22.04

### Description
A limited number of filter names cause exceptions when you attempt to fetch them.

### What I Did
For routine testing checked if any exceptions; I print these here.

```
filters = fpbase.list_filters()
dyes = fpbase.list_fluorophores()

for filter in filters:
    try:
        filter_object = fpbase.get_filter(filter)
    except:
        print(filter)

for dye in dyes:
    try:
        dye_object = fpbase.get_fluorophore(dye)
    except:
        print(dye)

the above code outputs:

chroma-50-50-beamsplitter
chroma-et750sp-2p8
crest-standard-penta-bandpass-dichroic
omega-1085alp
omega-1620-hsp-25.4
omega-1660-hsp-50.8
omega-380sp
omega-390sp-od4
omega-415sp
omega-420sp
omega-435sp
omega-450sp
omega-450sp-rapidedge
omega-455esp
omega-460sp-prefilter
omega-460sp-rapidedge
omega-470esp
omega-470sp
omega-470sp-rapidedge
omega-480sp-rapidedge
omega-485sp
omega-485spxr
omega-490aesp
omega-490sp
omega-490sp-rapidedge
omega-495sosp
omega-499sp
omega-500gsp
omega-500sp-rapidedge
omega-505sp
omega-510sp-rapidedge
omega-520sp-rapidedge
omega-530sp-rapidedge
omega-540sp
omega-540sp-prefilter
omega-540sp-rapidedge
omega-550asp
omega-550sp-rapidedge
omega-560sp-02
omega-560sp-rapidedge
omega-570sp-rapidedge
omega-580sp-rapidedge
omega-590sp-rapidedge
omega-595aesp
omega-600absp
omega-600sp-rapidedge
omega-601agsp-x
omega-610sp
omega-610sp-rapidedge
omega-620sp-rapidedge
omega-630sp
omega-630sp-rapidedge
omega-632aesp
omega-638aesp
omega-640efsp
omega-640sp
omega-640sp-rapidedge
omega-644aesp
omega-650asphit
omega-650gsp
omega-650sp-rapidedge
omega-660sp-rapidedge
omega-670sp-rapidedge
omega-678aesp
omega-680asp
omega-680sp-rapidedge
omega-690sp-rapidedge
omega-700cfsp
omega-700sp
omega-700sp-rapidedge
omega-715agsp
omega-720sp
omega-740cfsp-19mm
omega-748aesp
omega-750cfsp-50sq
omega-750gsp
omega-750sp
omega-791aesp
omega-800cfsp
omega-800efsp
omega-840sp
omega-850gsp
omega-900asp
omega-960sp
omega-990dcsp
omega-hmir
omega-hot-mirror
semrock-bsp01-785r
semrock-ff01-1010-sp
semrock-ff01-276-sp
semrock-ff01-300-sp
semrock-ff01-311-sp
semrock-ff01-390-sp
semrock-ff01-424-sp
semrock-ff01-440-sp
semrock-ff01-468-sp
semrock-ff01-492-sp
semrock-ff01-498-sp
semrock-ff01-533-sp
semrock-ff01-546-sp
semrock-ff01-612-sp
semrock-ff01-650-sp
semrock-ff01-670-sp
semrock-ff01-680-sp
semrock-ff01-715-sp
semrock-ff01-720-sp
semrock-ff01-745-sp
semrock-ff01-750-sp
semrock-ff01-758-sp
semrock-ff01-760-sp
semrock-ff01-770-sp
semrock-ff01-775-sp
semrock-ff01-790-sp
semrock-ff01-842-sp
semrock-ff01-890-sp
semrock-ff01-940-sp
semrock-ff01-945-sp
semrock-ff01-950-sp
semrock-ff02-694-sp
semrock-sp01-633ru
sp01-561ru-25
t685sp

```

Error for each of these seems the same, and Traceback is:

```
ValidationError: 2 validation errors for SpectrumResponse
data.spectrum.subtype
  Input should be 'A_2P', 'BM', 'BP', 'BS', 'BX', 'EM', 'EX', 'LP', 'PD', 'QE' or 'AB' [type=enum, input_value='SP', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum
data.spectrum.ownerFilter.spectrum.subtype
  Input should be 'A_2P', 'BM', 'BP', 'BS', 'BX', 'EM', 'EX', 'LP', 'PD', 'QE' or 'AB' [type=enum, input_value='SP', input_type=str]
    For further information visit https://errors.pydantic.dev/2.10/v/enum

```

**Comment by tlambert03** [comment]
Ha, 
I'm sorry I didn't check the full database on that last issue.

I'll do a comprehensive test later today and fix it up again.  Thanks for your patience

**Comment by JBeckwith** [comment]
It's no worries; I want to emphasise that this code is really good, and you're doing a lot better than me in speed of fixing issues!

**Comment by tlambert03** [comment]
fixed in #4, and I also ran a test to make sure that all names returned by `list_x` work when called with the corresponding `get_x` ... I don't want to run them every time, to be gentle on the fpbase servers :) 

do let me know if you run into anything else, or if you have any additional feature requests 👍
